config/services.yaml

```diff
@@ -9,3 +9,8 @@ services:
     neusta_converter.default_converter:
         abstract: true
         class: Neusta\ConverterBundle\Converter\DefaultConverter
+
+    neusta_converter.same_property_populator:
+        abstract: true
+        public: true
```

Comment:
Why does it need to be `public`? Afaik converters need to be public, but populators don't, right?

Reply:
My test was not running until I made it public.

Reply:
That's because in the test, you fetch it directly from the container. But apart from that, this won't be the case, as will be injected into the converter. So I think it should only be made `public` in the test config.